frontend/src/config/i18n/de/seeds.json

```
@@ -19,7 +19,7 @@
     "placeholder_binomial_name": "Tomate",
     "placeholder_origin": "Daheim",
     "placeholder_taste": "nussig",
-    "placeholder_yield": "1",
+    "placeholder_yield": "Hoch",
     "placeholder_price": "2,99€",
     "placeholder_generation": "0",
     "error_price_must_be_number": "Der Preis muss eine Zahl sein. z.B. 2,99",

```

I only get "please enter a number" and not this text? And the error can be improved:

```suggestion
    "error_price_must_be_number": "Entschuldigung, der Preis muss eine Zahl mit . als Komma sein. z.B. 2.99",

```

---

Or if we support ",", too:

```suggestion
    "error_price_must_be_number": "Entschuldigung, ich verstehe nur einen Preis als Zahl mit Punkt (.) oder Beistrich (,) als Komma, z.B. 2,99",
```